samples/models/createModel.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(
+  projectId = 'my_project',
+  datasetId = 'my_dataset',
+  modelId = 'my_model',
+) {
+  // [START bigquery_create_model]
+  // Import the Google Cloud client library
+  const {BigQueryClient} = require('@google-cloud/bigquery');
+  const bigquery = new BigQueryClient();
+
+  async function createModel() {
+    // Creates a model named "my_model" in "my_dataset".
+
+    /**
+     * TODO(developer): Uncomment the following lines before running the sample
+     */
+    // const datasetId = "my_dataset";
+    // const modelId = "my_model";
+
+    const query = `CREATE OR REPLACE MODEL \`${projectId}.${datasetId}.${modelId}\`
```

Comment:
should we use a lighter model like in the test ? This one takes a bit longer to create and reserve mode resources, not sure if is gonna become flaky with time in our CI/CD pipeline